docker-entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+set -e
+
+if [ $(echo "$1" | cut -c1) = "-" ]; then
+  echo "$0: assuming arguments for bitcoind"
+
+  set -- bitcoind "$@"
+fi
+
+if [ $(echo "$1" | cut -c1) = "-" ] || [ "$1" = "bitcoind" ]; then
+  mkdir -p "$BITCOIN_DATA"
+  chmod 700 "$BITCOIN_DATA"
+  chown -R bitcoin "$BITCOIN_DATA"
+
+  echo "$0: setting data directory to $BITCOIN_DATA"
+
+  set -- "$@" -datadir="$BITCOIN_DATA"
+fi
+
+if [ "$1" = "bitcoind" ] || [ "$1" = "bitcoin-cli" ] || [ "$1" = "bitcoin-tx" ]; then
```

Comment:
What would you think of removing all this and just always implying that we always execute bitcoind?

If I understood correctly, the part with "investigating the state" would look like this:

- Run this container
- `docker exec -it abcd /bin/bash`

Do stuff inside the container, like calling bitcoin-cli and so furth.

Do we ever _launch_ the container with anything different but `bitcoind`?

Reply:
Yeah if you really just wanna run run commands without bitcoind running just do --entrypoint=/usr/bin/bash IMO

Reply:
Happy to remove that yes.